cmd/handoff-bss-metadata.go

```diff
@@ -371,19 +371,21 @@ func buildPITArgs(base string) string {
 			cmdlineParts[i] = fmt.Sprintf("hostname=ncn-m001")
 		} else if strings.HasPrefix(part, "ifname=mgmt0") && len(macs) >= 2 {
 			cmdlineParts[i] = fmt.Sprintf("ifname=mgmt0:%s", macs[0])
-		} else if strings.HasPrefix(part, "ifname=mgmt1") && len(macs) >= 2 {
+		} else if strings.HasPrefix(part, "ifname=mgmt1") && len(macs) < 4 {
```

Comment:
For consistency I would break out the second conditional in 374 and 376 into a nested conditional like we do for mgmt2 and mgmt3.

For example:
```
		} else if strings.HasPrefix(part, "ifname=mgmt1") {
			if len(macs) >= 4 {
				cmdlineParts[i] = fmt.Sprintf("ifname=mgmt1:%s", macs[2])
			} else {
				cmdlineParts[i] = fmt.Sprintf("ifname=mgmt1:%s", macs[1])
			}
		}
```